helper/debug.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
good catch! I accidentally included it when I rebased. I've deleted the file from the commit.